pymc/sampling.py

```diff
@@ -2093,6 +2094,68 @@ def sample_prior_predictive(
     return pm.to_inference_data(prior=prior, **ikwargs)
 
 
+def draw(vars, draws=500, mode=None, **kwargs) -> Dict[str, np.ndarray]:
```

Comment:
We should just return the np.ndarray outputs directely, and not wrap them in a dictionary. That functionality is cover by prior_predictive

Reply:
Hi, so in case of drawing samples for several variables: instead of return Dict[str, np.ndarray], we will just return a list of np.ndarray right?

Reply:
Yes